.gitignore

```diff
@@ -15,3 +15,6 @@ pom.xml.asc
 *.tif
 *.xml
 *.txt
+*.~
```

Comment:
what are these for?

Reply:
Excluding emacs backups, e.g. project.clj~